modules/static/grpc-ecosystem/grpc-gateway/initref

```diff
@@ -1 +1 @@
-813bbcd1a729fb6548d3a02ce50c86f2278cb5c0
+v2.20.0
```

Comment:
This is the latest release: https://github.com/grpc-ecosystem/grpc-gateway/releases

The closest release to the original initref commit would have been:

```
➜  grpc-gateway git:(813bbcd1) git describe --all 813bbcd1a729fb6548d3a02ce50c86f2278cb5c0
tags/v2.11.3-60-g813bbcd1
```